### ISSUE TITLE
Enabling multiple addons on MicroK8s in one invocation may lead to unreachable API server

### ISSUE THREAD

**Issue by worldofgeese** [garden-io/garden#4860]
### Current Behavior

When running `microk8s enable addons dns storage registry ingress`, a `WARNING` is printed:

```python
            "WARNING: Do not enable or disable multiple addons in one command.\n"
            "         This form of chained operations on addons will be DEPRECATED in the future.\n"
            f"         Please, {action} one addon at a time: 'microk8s {action} <addon>'"
```

The responsible line of code is [linked here](https://github.com/canonical/microk8s/blob/07e7ed6f9aa83b986cebc3667588435463dab13d/scripts/wrappers/common/utils.py#L531).

### Expected behavior

Garden should enable addons single invocation by single invocation.

Impacted code in our CircleCI [config.yml](https://github.com/garden-io/garden/blob/69f22b7a972f6f6a327080e947520f655eeacf8b/.circleci/config.yml#L635) and our [config.ts](https://github.com/garden-io/garden/blob/69f22b7a972f6f6a327080e947520f655eeacf8b/core/src/plugins/kubernetes/local/config.ts#L137).
 
### Reproducible example

Use https://github.com/garden-io/quickstart-example

### Workaround

Manually enable addons as user before running first `garden deploy`.

### Additional context

Following `garden deploy` auto-enabling all addons on Windows 21H2, the MicroK8s API server became unreachable, requiring a complete VM delete and re-creation. I manually enabled the required addons one-by-one before running `garden deploy` again, at which point MicroK8s was accessible.

[garden-microk8s-error-log.log](https://github.com/garden-io/garden/files/12093271/garden-microk8s-error-log.log)

### Your environment

* **OS:** Windows Enterprise LTSC 21H2
* **How I'm running Kubernetes:** MicroK8s

`garden version` 0.13.8

**Comment by vvagaytsev** [comment]
@worldofgeese, thanks for reporting this! It should be an easy-to-fix issue. I'm taking a look now.

By the way, the [link to the config.ts](https://github.com/garden-io/garden/blob/69f22b7a972f6f6a327080e947520f655eeacf8b/core/src/plugins/kubernetes/local/config.ts#L137) points to the `minikube`-specific piece of code, not to the `microk8s`. So I think we should fix the [function `configureMicrok8sAddons`](https://github.com/garden-io/garden/blob/69f22b7a972f6f6a327080e947520f655eeacf8b/core/src/plugins/kubernetes/local/microk8s.ts#L56-L59).

Should we use sequential addon installation for `minikube` as well?

**Comment by vvagaytsev** [comment]
Thanks @worldofgeese! Could you try [this build](https://app.circleci.com/pipelines/github/garden-io/garden/19169/workflows/64f4fdf2-70af-4628-a3b7-712c16017e73/jobs/350538/artifacts), please?

**Comment by vvagaytsev** [comment]
@worldofgeese the fix has been released in [0.13.9](https://github.com/garden-io/garden/releases/tag/0.13.9), please try it out.
